src/components/achievementsModal.html

```diff
@@ -10,8 +10,45 @@ <h5 data-bind="tooltip: { title: 'As you earn achievements, you gain up to 100%
             </div>
 
             <div class="modal-body table-responsive p-0">
+                <div class="tab-content p-3">
+                    <div class="tab-pane active">
+                        <div class="text-left">
+                            <div class="form-row">
+                                <div class="form-group col-md-6 col-6">
+                                    <label>Status</label>
+                                    <select class="custom-select" onchange="AchievementHandler.getAchievementOption(this.value, AchievementHandler.filter.type())">
+                                        <option value="all" selected>All</option>
+                                        <option value="false">Uncompleted</option>
+                                        <option value="true">Completed</option>
+                                    </select>
+                                </div>
+                                <div class="form-group col-md-6 col-6">
+                                    <label>Type</label>
+                                    <select class="custom-select" onchange="AchievementHandler.getAchievementOption(AchievementHandler.filter.status(), this.value)">
+                                        <option value="all" selected>All</option>
```

Comment:
We could use a knockout foreach binding here to save us having to update in the future with new achievement types.

There is an example of using this with an enum in `src/components/shipModal.html`
```html
<div style='margin-top:20px' data-bind="foreach: GameHelper.enumStrings(GameConstants.Region)">
```
and in the same place, a nice utility function to turn the enum key into something better to display
```js
text: GameConstants.camelCaseToString($data)
```

```js
>> GameConstants.camelCaseToString("itemDiggedUp")
<- "Item Digged Up"
```

Reply:
ok, I will do that